packages/KIcon/icons/icn-team-member.svg

```diff
@@ -0,0 +1,5 @@
+<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 30 30">
+    <title>Team Member</title>
+    <path d="M0 15C0 6.716 6.716 0 15 0c8.284 0 15 6.716 15 15 0 8.284-6.716 15-15 15-8.284 0-15-6.716-15-15Z" fill="#0A7FAE"/>
```

Comment:
```suggestion
    <path d="M0 15C0 6.716 6.716 0 15 0c8.284 0 15 6.716 15 15 0 8.284-6.716 15-15 15-8.284 0-15-6.716-15-15Z" fill="#A3BBCC"/>
```

The `fill` color needs to be updated to `#A3BBCC` so that all icons render as the same color by default.

![image](https://user-images.githubusercontent.com/2229946/156637140-49083e2a-0395-4836-8e22-f171ed758ed5.png)

Reply:
Fixed